.gitignore

```diff
@@ -39,3 +39,4 @@ terraform.tfstate
 
 .openapi-generator/
 internal/service/*/.openapi-generator-ignore
+internal/.DS_Store
```

Comment:
should be just `.DS_Store` as it can occur in any directory. Also, please add a new line to separate from other groupings

Reply:
done